dipy/reconst/tests/test_dti.py

```diff
@@ -783,9 +783,10 @@ def test_min_signal_alone():
     ten_model = dti.TensorModel(gtab)
     fit_alone = ten_model.fit(data[idx])
     fit_together = ten_model.fit(data)
-    npt.assert_almost_equal(fit_together.model_params[idx],
-                            fit_alone.model_params)
 
+    npt.assert_almost_equal(fit_together.model_params[idx],
```

Comment:
I believe that assert_almost_equal has a tolerance of seven digits for default. Why are we requiring higher accuracy here?

Reply:
On Windows, this test failed sometimes because of a difference after 14 digits. So, Are you sure `assert_almost_equal` has this tolerance?
Furthermore, [it is already there on the master](https://github.com/nipy/dipy/blob/master/dipy/reconst/tests/test_dti.py#L785) so I do not know why it disappears. 😄

Reply:
At least that's what the docs say... https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.testing.assert_almost_equal.html

Reply:
ok, I removed this parameter.